tests/php/Unit/Internal/Domain/PhelFnNormalizerTest.php

```diff
@@ -20,17 +31,16 @@ public function test_group_key_one_function(): void
         ]);
 
         $normalizer = new PhelFnNormalizer($phelFnLoader);
-        $actual = $normalizer->getGroupedFunctions();
+        $actual = $normalizer->getPhelFunctions();
 
         $expected = [
-            'fn-name' => [
-                PhelFunction::fromArray([
-                    'fnName' => 'fn-name',
-                    'doc' => '',
-                    'fnSignature' => '',
-                    'desc' => '',
-                ]),
-            ],
+            PhelFunction::fromArray([
+                'fnName' => 'fn-name',
```

Comment:
@Chemaclass do we have to update also the webpage project?

Reply:
Yes, I have already the code prepared. I will create a PR on the web repo once this PR is merged.